R/infillOptFocus.R

```diff
@@ -8,11 +8,35 @@
 # See infillOptCMAES.R for interface explanation.
 infillOptFocus = function(infill.crit, models, control, par.set, opt.path, design, iter, ...) {
   global.y = Inf
+  
+  allRequirements = extractSubList(par.set$pars, "requires", simplify = FALSE)
+  allUsedVars = unique(do.call(base::c, sapply(allRequirements, all.vars)))
```

Comment:
What is `all.vars` in this context? I feel stupid, but I cannot find it.
I also find it strange to use `do.call(base::c, ...)`. Won't `unlist` do the trick?